docs/ref/models/querysets.txt

```diff
@@ -2030,6 +2030,37 @@ See the :doc:`/topics/db/sql` for more information.
   filtering. As such, it should generally be called from the ``Manager`` or
   from a fresh ``QuerySet`` instance.
 
+``comment()``
+~~~~~~~~~~~~~
+
+.. versionadded:: 5.2
+
+.. method:: comment(message)
+
+Adds an SQL comment to be inserted into the resultant SQL statement, after
+``SELECT``/``UPDATE``, and if applicable, ``DISTINCT``.. For example::
```

Comment:
Is it not possible to make it work for `INSERT`, at least when using `Model.objects.comment(...).create(...)` ?